docs/FAQ.md

```diff
@@ -44,3 +44,37 @@ or client side to unmarshal a response) the default value is used to set the
 value of missing fields. Note that if the attribute is required then the
 generated code returns an error if the corresponding field is missing. So this
 only applies for non required attributes with default values.
+
+# How are views for a result type computed?
+
+Views can be defined on a result type. If a method returns a result type with
```

Comment:
Would be good to start with the user facing consequences of defining multiple views, that is the fact that the service method returns an extra value in this case which corresponds to the name of the view that should be rendered in the response. Also would be good to have some examples of design and corresponding generated code.